lib/sentry/check_in.ex

```diff
@@ -127,6 +128,7 @@ defmodule Sentry.CheckIn do
         failure_issue_threshold: number_schema_opts,
         recovery_threshold: number_schema_opts,
         timezone: [type: :string],
+        owner: [type: :string],
```

Comment:
Can we say something like this in the docs for this type:

```
The type for the check-in struct.

`:owner` is available since v10.10.0.
```

Reply:
good call. I extracted the config into its own type to make it clearer